lib/shopify_api/webhooks/registrations/http.rb

```diff
@@ -9,12 +9,12 @@ class Http < Registration
 
         sig { override.returns(String) }
         def callback_address
-          if @path.match?(%r{^https://})
```

Comment:
Just an observation, not about your PR, but it's odd that we call this `@path` when it seems to be a URI.

Reply:
ya, as I worked on this I noticed a lot of inconsistencies like that 🤦